cloudbuild.yaml

```diff
@@ -12,3 +12,34 @@ steps:
         fi
     env:
       - "TAG_NAME=${TAG_NAME}"
+
+  - id: deploy-csharp
+    name: "bitnami/dotnet-sdk"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        if [ "$TAG_NAME"  ] ; then
+          cp vmlapis.csproj gen/csharp
+          cd gen/csharp
+          dotnet pack -o . -c Release
+          dotnet nuget push vmlapis.$TAG_NAME.nupkg --source https://api.nuget.org/v3/index.json --api-key $$NUGET_API_KEY
+        fi
+    secretEnv: ['NUGET_API_KEY']
+
+  - id: deploy-java
+    name: "maven:3.6.3-jdk-8"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        if [ "$TAG_NAME" ] ; then
+          cp -R gen/java java-package/src/main/java
+          cd java-package
+          mvn deploy
+        fi
+
+availableSecrets:
+  secretManager:
+    - versionName: projects/161107618804/secrets/nuget-api-key/versions/latest
```

Comment:
This key has maximum validity for one year and then it needs to be regenerated on nuget.org. I've seen some rotation and expiry settings in Secret Manager. Perhaps it is desirable to adjust them. Right now the key is only set to expire 364 days from now which would rend this inoperable.

Reply:
True, but if it expires on nuget, I doubt google cloud can somehow automatically take that token from nuget after a year. I'd suggest we'd set up a reminder after 11-11,5 months to make a new api key on nuget. Or perhaps we can set up a notification from nuget when the token expires.

Reply:
That's what the rotation settings are from what I've understood. They don't rotate the key, but rather can send out Pub/Sub for example and other things, but didn't study it much further

Reply:
Mm... if that's the case, it would be worth investigating if we can make this automatic somehow. A simple notification to the team would do just fine for now though, I believe.